paper/paper.md

```diff
@@ -125,33 +127,33 @@ bibliography: paper.bib
 # Summary
 
 VIVO [Pronunciation: vee-voh] is member-supported, enterprise open source 
-software and an ontology for representing scholarship.  VIVO supports recording, editing, 
+software and an ontology for representing scholarship. VIVO supports recording, editing, 
 searching, browsing and visualizing scholarly activity. VIVO encourages research 
-discovery, expert finding, network analysis and assessment of research impact.  
-VIVO is easily extended to support additional domains of scholarly activity.
+discovery, expert finding, network analysis and assessment of research impact. 
+VIVO is easily extended to support additional domains of scholarly activity [@borner_vivo:_2012].
```

Comment:
These `[@xxx]` references do not appear to resolve as links. @hauschke , were you expecting a different behavior?

Reply:
Although you appear to be aligned with the recommendation:
https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html

@hauschke , as far as you know, is there any way of testing the references?

Reply:
to test, change the .md or .pdf, then put a comment in the review issue with ```@whedon generate pdf``` - editors and authors can do this, which might be an issue here.

Reply:
@danielskatz : I am not sure I follow your suggestion. Can you re-articulate what you are saying?

Reply:
To regenerate the PDF (to test changes made in the .md and .bib files), in https://github.com/openjournals/joss-reviews/issues/1182 an editor or the author can put a new comment in that contains ```@whedon generate pdf``` - this will tell @whedon to regenerate the PDF from the md and  bib.